types/frisby/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
please remove this lint exception, thanks~

Reply:
Fixed. Thank you.

Also,  `FrisbyResponse` class is not directly referenced, so I changed it to `interface`.